shielddash/settings.py

```diff
@@ -27,7 +27,7 @@
 SECRET_KEY = config('SECRET_KEY', default='change me to a real secret key')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = config('DEBUG', default=False, cast=bool)
+DEBUG = config('DEBUG', default=True, cast=bool)
```

Comment:
I prefer defaults more towards a production and secure server. But also don't feel strongly about it. I can see being more contributor friendly also.